web-src/src/services/RemoteFavoritesService.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+import { wretchRetry } from '../helpers/NetworkHelper.js';
+import { AppError } from '../helpers/ErrorMapper.js';
+
+export class RemoteFavoritesService {
+  constructor({
+    favoritesEndpoint,
+    imsOrg,
+    accessToken,
+    websiteUrl,
+  }) {
+    this.favoritesEndpoint = favoritesEndpoint;
+    this.imsOrg = imsOrg;
+    this.accessToken = accessToken;
+    this.websiteUrl = websiteUrl;
+
+    console.debug(`Favorites Endpoint: ${this.favoritesEndpoint}`);
+  }
+
+  async get_favorites() {
```

Comment:
Could we consider using camel case for naming methods? Using methods with underscores is unconventional in JavaScript, so we'd need to make an exception in our linting rules.